Sources/ArcGISToolkit/Components/FloorFilter/SiteAndFacilitySelector.swift

```diff
@@ -117,7 +118,7 @@ struct SiteAndFacilitySelector: View {
                     tag: site,
                     selection: Binding(
                         get: {
-                            return shouldAutoSelect ? viewModel.selectedSite : nil
+                            userBackedOutOfSelectedSite ? nil : viewModel.selectedSite
```

Comment:
Now, the binding will only return `nil` if the user tapped the back arrow, meaning we won't visually loose the selection as a result of a pan gesture.